pkg/cluster/k8sres.go

```diff
@@ -222,6 +223,9 @@ PatroniInitDBParams:
 	if patroni.Slots != nil {
 		config.Bootstrap.DCS.Slots = patroni.Slots
 	}
+	if patroni.SynchronousMode != false {
```

Comment:
```suggestion
	if patroni.SynchronousMode {
```

As there is no default, this field will be `nil` when left out. So we should only enter this code block when `true`. Right now, I'm not 100% sure if we have to check `patroni.SynchronousMode != nil`, too